### PR TITLE
chore: rename package

### PR DESCRIPTION
**Pull request by rolznz**
this is a quite radical name change so we also should update the repository too, I think.

Follow up steps:
- [ ] Deprecate old npm package
- [ ] Rename github repo
- [ ] Announce package changes in Alby blog

### CODE REVIEW

**Comment by pavanjoshi914** [comment]
isn't this a breaking change for existing alby-js-sdk user? :thinking: if yes probably we shall notify user's of package before taking this step

**Comment by rolznz** [comment]
@pavanjoshi914 I think they just won't receive updates until they change their dependency.

**Comment by pavanjoshi914** [comment]
cool! I see  ```npm decpricate```  is quite helpfull here

File: src/window.js [hunk 1]
```diff
@@ -1,3 +1,8 @@
...
+// assign alby lightning exports to global window object (for index.browser.js)
+
+// @deprecated
 // @ts-ignore
-window["albyTools"] = require('./index.cjs');
\ No newline at end of file
+window["albyTools"] = require('./index.cjs');
+
+// @ts-ignore
+window["albyLightning"] = require('./index.cjs');
```
Comment (bumi) [inline src/window.js:8]: we should use the new name here I guess.
window.lightningTools ? 

if the package is named lightning it's window.lightning ?

Comment (reneaaron) [reply src/window.js:8]: why do we have those window variables in there at all? :thinking: 

for being able to use it via CDN includes?

Comment (rolznz) [reply src/window.js:8]: @reneaaron yes. e.g. using alby-tools in a simple codepen.

**Comment by reneaaron** [comment]
Closing in favor of #46

**Pull request closed without merging**